assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -268,6 +293,28 @@ const baseReducer = ( state, { type } ) => {
 			};
 		}
 
+		case TRANSFORM_AND_SORT_ACCOUNT_SUMMARIES: {
+			const accountSummaries = [ ...state.accountSummaries ].map(
+				( account ) => {
+					const obj = appendAccountID( account, 'account' );
+					obj.propertySummaries = account.propertySummaries.map(
```

Comment:
This broke for me when testing on an account with no properties (calling `map` on `undefined`).